test/assert_functions.ts

```diff
@@ -0,0 +1,54 @@
+import * as assert from 'assert';
+
+import * as JSON5 from 'json5';
+
+import { shell } from './test_functions';
+
+export function assertNodePackageVersion(version: string): void {
+  assert.strictEqual(getNodePackageVersion(), version);
+}
+
+export function assertTagCount(count: number): void {
+  const tags = getGitTags();
+
+  assert.strictEqual(tags.length, count);
+}
+
+export function assertTagExists(tag: string): void {
+  const tags = getGitTags();
+
+  assert.ok(tags.includes(tag), `Expected tag '${tag}' to exist, got: ${JSON.stringify(tags)}`);
+}
+
+export function rejectTagExists(tag: string): void {
```

Comment:
```suggestion
export function assertTagDoesNotExist(tag: string): void {
```

Wirkt konsistenter, wenn die Funkion hier dem gleichen Namensschema folgt.

Reply:
Kann ich gerne ändern.

P.S. Wenn ich da jetzt auf "Commit suggestion" drücke, explodiert mein Build ...